doc/subsystems/bluetooth/index.rst

```diff
@@ -72,8 +76,8 @@ Bluetooth stack:
 Qualification Testing
 *********************
 
-PICS Features
-=============
+-PICS Features
+-=============
```

Comment:
What's going on here? I.e. what are those `-` added to the beginning of the lines?

Reply:
sharp eyes, no idea how this happened.